app/javascript/components/legend.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
```

Comment:
The filename wasn't changed to `Legend.jsx`

Reply:
Thats weird. Its totally changed and updated. 
<img width="838" alt="screen shot 2018-02-27 at 9 43 18 am" src="https://user-images.githubusercontent.com/6424683/36744868-bc1db262-1ba2-11e8-91bb-4e5ab2c74a0a.png">

Reply:
Timestamp for legend says 9:15 too.